### PR TITLE
Add HEVC muxing support for CMAF

### PR DESCRIPTION
**Pull request by gBillal**

### CODE REVIEW

File: lib/membrane_mp4/muxer/cmaf/track_samples_queue.ex [hunk 1]
```diff
@@ -326,4 +326,8 @@ defmodule Membrane.MP4.Muxer.CMAF.TrackSamplesQueue do
         excess_samples: []
     }
   end
+
+  defp key_frame?(%{metadata: %{h264: %{key_frame?: true}}}), do: true
+  defp key_frame?(%{metadata: %{h265: %{key_frame?: true}}}), do: true
+  defp key_frame?(_sample), do: false
```
Comment (mat-hek) [inline lib/membrane_mp4/muxer/cmaf/track_samples_queue.ex:332]: Seems like this kind of function is repeated many times, maybe let's add a `codec_helper.ex` and put it there? I guess it should accept just the metadata to be most flexible.

**Review by mat-hek — COMMENTED** [review]
Awesome 😎 one thing and we can merge

File: lib/membrane_mp4/muxer/cmaf/track_samples_queue.ex (hunk 1 above)
Comment (gBillal) [reply lib/membrane_mp4/muxer/cmaf/track_samples_queue.ex]: I added the function to the already existing `Membrane.MP4.Helper` module.

**Review by mat-hek — APPROVED** [review]

**Comment by mat-hek** [comment]
@gBillal next time don't forget to refresh my review, otherwise I don't know the PR's ready ;)

**Pull request merged**